alicloud/resource_alicloud_gpdb_instance.go

```diff
@@ -120,39 +120,7 @@ func resourceAlicloudGpdbInstanceRead(d *schema.ResourceData, meta interface{})
 }
 
 func resourceAlicloudGpdbInstanceCreate(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*connectivity.AliyunClient)
-	gpdbService := GpdbService{client}
-
-	request, err := buildGpdbCreateRequest(d, meta)
-	if err != nil {
-		return WrapError(err)
-	}
-	var raw interface{}
-	err = resource.Retry(5*time.Minute, func() *resource.RetryError {
-		raw, err = client.WithGpdbClient(func(client *gpdb.Client) (interface{}, error) {
-			return client.CreateDBInstance(request)
-		})
-		if err != nil {
-			if IsExpectedErrors(err, []string{"SYSTEM.CONCURRENT_OPERATE"}) {
-				return resource.RetryableError(err)
-			}
-			return resource.NonRetryableError(err)
-		}
-		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
-		return nil
-	})
-	response, _ := raw.(*gpdb.CreateDBInstanceResponse)
-	if err != nil {
-		return WrapErrorf(err, DefaultErrorMsg, "alicloud_gpdb_instance", request.GetActionName(), AlibabaCloudSdkGoERROR)
-	}
-	d.SetId(response.DBInstanceId)
-
-	stateConf := BuildStateConf([]string{"Creating"}, []string{"Running"}, d.Timeout(schema.TimeoutCreate), 10*time.Minute, gpdbService.GpdbInstanceStateRefreshFunc(d.Id(), []string{"Deleting"}))
-
-	if _, err := stateConf.WaitForState(); err != nil {
-		return WrapErrorf(err, IdMsg, d.Id())
-	}
-	return resourceAlicloudGpdbInstanceUpdate(d, meta)
+	return Error("Sorry, creating new reserved storage mode instance is no longer supported since v1.127.0, you can use alicloud_gpdb_elastic_instance to create a flexible storage mode instance instead.")
```

Comment:
If the OpenAPI does not offline, there can not offline this resource. You can add the statement in its docs by a NOTE